server/svix-server/src/v1/endpoints/event_type.rs

```diff
@@ -191,7 +191,7 @@ async fn create_event_type(
 async fn get_event_type(
     Extension(ref db): Extension<DatabaseConnection>,
     Path(evtype_name): Path<EventTypeName>,
-    permissions: Permissions,
+    AuthenticatedOrganization { permissions }: AuthenticatedOrganization,
```

Comment:
OK, the naming is really confusing, we need to do a better job at naming these three Authenticated variants. Any suggestions?

Reply:
Idea:
* AuthenticatedOrganization - keep it.
* AuthenticatedApplication - keep it.
* AuthenticatedOrganizationWithApplication - instead of OrganizationAuthentiacedApplication

Yeah, it's a bit long, but it's only used standalone on its own line in parameters list so maybe not a biggie. What do you think?